packages/neuron-ui/src/locales/en.json

```diff
@@ -309,7 +309,9 @@
       "copy-balance": "Copy Balance",
       "copy-address": "Copy Address",
       "create": "Create {{name}} Asset Account",
-      "destroy": "Destroy {{name}} Asset Account"
+      "destroy": "Destroy {{name}} Asset Account",
+      "confirmation": "{{confirmationCount}} Confirmation",
+      "confirmations": "{{confirmationCount}} Confirmations"
```

Comment:
Looks like the same meaning as `"confirming-with-count": "{{confirmations}} Confirmations"`?

Reply:
I mean if we use one variable to handle this value, it will be `1 Confirmations`, which may not be acceptable.